code/modules/ninja/suit/n_suit_verbs/energy_net_nets.dm

```diff
@@ -44,6 +44,12 @@ It is possible to destroy the net by the occupant or someone else.
 		qdel(src)//Get rid of the net.
 		return
 
+	if(ishuman(affecting))
+		var/mob/living/carbon/human/M = affecting
+		if(M.staminaloss <= 99 || M.stat != UNCONSCIOUS)
```

Comment:
```suggestion
		if(M.staminaloss <= 99 && M.stat != UNCONSCIOUS)
```
would be way more appropriate, otherwise you won't even be able to capture a conscious person in stamcrit

Also, I'm not entirely sure about the need to keep the person in stamcrit the entire time just to not make the net instantly *poof* away. I've buffed the gloves a little bit a while ago, to apply some stun, but I'm not even sure if a ninja would be able to capture anyone like this on one battery charge.